README.md

```diff
@@ -1,129 +1,199 @@
 # JavaScript Variables
 
 ## Overview
-
-In this lesson, we'll introduce variables and how to declare them in Javascript. 
+In this lesson, we'll introduce JavaScript variables and explain how to declare, assign, and access them.
 
 ## Objectives
+1. Understand what a variable is.
+2. Describe how to name variables in JavaScript.
+3. Use `var` to create and access variables in pre-ES2015 code.
+4. Explain the basics of the variable lifecycle — the distinct declaration and assignment phases.
+5. Define when to use `const`, `let`, and `var` for declaring variables.
 
-1. Explain how to declare a variable in JavaScript
-2. Explain what variables are good for
-3. Declare a variable without assigning a value
-4. Declare and define a variable
-5. Explain multi-line variable assignment
-
-## What are Variables?
+## What is a variable?
+A variable is a container in which we can store values for later retrieval.
 
-Sometimes, we need to store a piece of information so that we can refer to it — often multiple times — later on. In real life, we would probably jot down a note in a notebook or on a post-it.
+Imagine a box that can hold any type of data: a number, string, boolean, object — even an `undefined`. We take some data that we want to store, place it inside the box, and hand the box off to the JavaScript engine, which stores it in memory. All done! Our data is safely cached until we need to access it again.
 
-![note](https://i.chzbgr.com/full/4950590208/h52A02E59/)
+![Raiders of the Lost Ark warehouse](https://user-images.githubusercontent.com/17556281/28639657-fea1930a-7216-11e7-8c38-45bc9fab96a7.gif)
 
-In programming, we use _variables_. Variables in JavaScript are used to store data that will be used in our program. A variable can point to almost any type of value including numbers, strings, arrays, objects, and functions.
+But wait! When we ask for the data back, how will the JavaScript engine know _which_ box to retrieve? We need to assign a name to our variable — a label for our box — so that we can tell the engine exactly which piece of stored data we want to access.
 
-Variables are assigned values using the = operator. Variable names are typically all lower case; in the case of multiple words, the words are joined together using lowerCamelCase.
+## Naming variables
+If you're interested in an exhaustive discussion of what constitutes a valid variable name in JavaScript, [have fun][valid variable names]. For almost every situation, follow these three rules, and you'll be fine:
```

Comment:
Tell them not to look at that unless you're super bored. I want to focus the student's attention